packages/btc/src/api/sendBtc.ts

```diff
@@ -18,31 +19,23 @@ export async function createSendBtcBuilder(props: SendBtcProps): Promise<{
   feeRate: number;
   fee: number;
 }> {
-  const tx = new TxBuilder({
+  // By default, all outputs in the sendBtc() API are fixed
+  const outputs = props.tos.map((to) => ({
+    fixed: true,
+    ...to,
```

Comment:
If the `to` contains a `fixed` parameter, and the value is false, then fixed will eventually be false

Reply:
Yes, this is intentional, allowing users to alter the default value of "fixed" if needed.